src/mupen64plus-video-gliden64.mk

```diff
@@ -128,13 +128,15 @@ ifeq ($(TARGET_ARCH_ABI), armeabi-v7a)
     # Use for ARM7a:
     MY_LOCAL_SRC_FILES += $(SRCDIR)/3DMathNeon.cpp.neon
     MY_LOCAL_SRC_FILES += $(SRCDIR)/gSPNeon.cpp.neon
+    MY_LOCAL_SRC_FILES += $(SRCDIR)/RSP_LoadMatrixNeon.cpp.neon
```

Comment:
I'm afraid that .neon extension will exclude these files from build. Sure, need to check it.